common/test/test_backintime.py

```diff
@@ -155,7 +155,9 @@ def test_local_snapshot_is_successful(self):
 
         line_contains_to_exclude = [
             "Gtk-WARNING",
-            "qt.qpa.plugin: Could not find the Qt platform plugin"
+            "qt.qpa.plugin: Could not find the Qt platform plugin",
+            'qt.dbus.integration: Could not connect "org.freedesktop.IBus" '
```

Comment:
This error message appeared in the output of my machine.
IBus is related to the setup of input method (switching between German and Japanese keyboard).